### PR TITLE
Remove unicode literals syntax from python2

### PR DESCRIPTION
**Pull request by lucc**
In order to confuse new developers less.

### CODE REVIEW

File: alot/helper.py [hunk 1]
```diff
@@ -179,7 +179,7 @@ def shorten_author_string(authors_string, maxlength):
         au = authors.pop()
         if len(au) > 1 and (remaining_length == 3 or (authors and
                                                       remaining_length < 7)):
-            authors_chain.appendleft(u'\u2026')
```
Comment (pazz) [inline alot/helper.py]: is this still necessary then? I presume that source files are utf-8 encoded?

File: tests/commands/test_global.py [hunk 2]
```diff
@@ -84,12 +84,12 @@ def test_set_gpg_sign_by_default_but_no_key(self):
...
 
     def test_get_template_decode(self):
-        subject = u'This is a täßϑ subject.'
-        to = u'recipient@mail.com'
-        _from = u'foo.bar@mail.fr'
-        body = u'Body\n地初店会継思識棋御招告外児山望掲領環。\n€mail body €nd.'
+        subject = 'This is a täßϑ subject.'
+        to = 'recipient@mail.com'
+        _from = 'foo.bar@mail.fr'
+        body = 'Body\n地初店会継思識棋御招告外児山望掲領環。\n€mail body €nd.'
```
Comment (pazz) [inline tests/commands/test_global.py]: compare this with the comment above, where we still use escape codes

**Comment by lucc** [comment]
I only found 3 cases.

**Comment by pazz** [comment]
Thanks for this!

**Pull request merged**